### PR TITLE
Remove MoPub

### PR DESCRIPTION
**Pull request by victor-takai**

### CODE REVIEW

File: nimbus-ios-sample.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -197,11 +195,10 @@
 		4D8A3FA52742C74B00201485 /* Mediation Platforms */ = {
 			isa = PBXGroup;
 			children = (
+				4DE1FE4D27FF58C2002C41BC /* MediationAdType.swift */,
+				4DE1FE4B27FF58C2002C41BC /* MediationIntegrationType.swift */,
+				4DE1FE4C27FF58C2002C41BC /* MediationViewController.swift */,
 				4DDD7A54274470760075FDAE /* Google */,
```
Comment (brugg) [inline nimbus-ios-sample.xcodeproj/project.pbxproj:201]: nit: not related to these changes, but just for consistency, can you rename `Google` to `GAM`?

File: nimbus-ios-sample/Mediation Platforms/MediationViewController.swift [hunk 2]
```diff
@@ -31,10 +31,7 @@ final class MediationViewController: DemoViewController {
     }()
     
     private var dataSource: [DemoDataSource<MediationIntegrationType, MediationAdType>] {
-        [
-            DemoDataSource(type: .google, values: MediationAdType.allCases),
-            DemoDataSource(type: .moPub, values: MediationAdType.allCases)
-        ]
+        [DemoDataSource(type: .google, values: MediationAdType.allCases)]
```
Comment (brugg) [inline nimbus-ios-sample/Mediation Platforms/MediationViewController.swift:34]: Same here. Change `google` to `gam`

**Review by brugg — APPROVED** [review]

File: nimbus-ios-sample/Config/secrets.json [hunk 3]
```diff
@@ -5,8 +5,6 @@
     "facebook_native_placement_id": "",
     "facebook_interstitial_placement_id": "",
     "facebook_banner_placement_id": "",
-    "mopub_banner_id": "",
```
Comment (inderdhir) [inline nimbus-ios-sample/Config/secrets.json:8]: Also remove the parsing from the ruby script for these two

**Review by inderdhir — APPROVED** [review]

File: nimbus-ios-sample/Config/secrets.json (hunk 3 above)
Comment (victor-takai) [reply nimbus-ios-sample/Config/secrets.json:8]: which ruby script?

**Pull request merged**